### ISSUE TITLE
Links that go to the same place 

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1430]
On the record detail page, there are many examples of metadata links with the same text that appear to go to the same page. For example, [this record](https://earthworks.stanford.edu/catalog/stanford-nj178qz9304) has the same creator and publisher - ML InfoMap (Firm). It also has overlapping subjects and themes. 

<img width="573" alt="Screenshot 2024-10-08 at 4 47 44 PM" src="https://github.com/user-attachments/assets/4a30b181-eb09-4ce3-878c-306bda0371fc">

SiteImprove flags it as "links that go to the same place," which is inaccurate. The creator link goes to a [faceted search where **_creator_** equals ML InfoMap (Firm)](https://earthworks.stanford.edu/?f%5Bdct_creator_sm%5D%5B%5D=ML+InfoMap+%28Firm%29) and the publisher link goes to a [faceted search where **_publisher_** equals ML InfoMap (Firm)](https://earthworks.stanford.edu/?f%5Bdct_publisher_sm%5D%5B%5D=ML+InfoMap+%28Firm%29). 

Once you [login to SiteImprove](https://siteimprove.stanford.edu/), go to - https://my2.siteimprove.com/Accessibility/1064059/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&pageSegments=&ruleName=sia-r81&ruleId=81&issueKind=2&exceptTags=1,2&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2 to see the full list of these issues (1,662 pages/3294 occurrences). 

However, there is a separate valid accessibility issue with links that have the same text that go to different places. A common example is "View more" and it's seen on different sections, such as News and Events. In that case, we add a clearer aria-label, such as "View more news" or "View more events." The aria-label should contain the same text as the original text link plus text that makes it clearer. 

**Guidelines:** 
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8
https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-link-only

I recommend that we add the field label, such as "ML InfoMap (Firm), Creator" and ML InfoMap (Firm), Publisher," or to be even clearer, we could try, "Search by ML InfoMap (Firm), Creator" and "Search by ML InfoMap (Firm), Publisher."

**Comment by thatbudakguy** [comment]
This might be tricky to balance with asking an SR to read the page straight through. If we add the field name to the accessible label of the facet value, it might end up reading something like: "Subject: Human Settlements (subject), Villages (subject), Census (subject), Demography (subject)..." etc. This would be exacerbated even further if we added the text "search for" or similar.

Needs some testing to come up with the right solution.

**Comment by kimdurante** [comment]
This might be resolved as we reaccession the data (we decided to do this during the last work cycle in order to move the files out of zips.)
In the past, we included the data creator and data provider (publisher) names in the metadata. However, more institutions are defaulting to just using the publisher name and not duplicating it in the creator field.
As I am going through our collections and reaccessioning data I am removing the creator field if it is the same as the publisher. So these should start to appear less and less.

I'm about halfway through this work, btw.

**Comment by corylown** [comment]
This impacts any blacklight app where we're displaying facets as links where there could be duplicate values in different fields.

**Comment by thatbudakguy** [comment]
Opened https://github.com/projectblacklight/blacklight/issues/3427 to track this in blacklight.
